zproject/settings.py

```diff
@@ -496,6 +496,11 @@ def get_secret(key):
     # can query using ./manage.py print_initial_password
     INITIAL_PASSWORD_SALT = get_secret("initial_password_salt")
 
+# Use best password hashing algorithm argon2 for PRODUCTION
+if PRODUCTION:
+    PASSWORD_HASHERS = ('django.contrib.auth.hashers.Argon2PasswordHasher',
+                        'django.contrib.auth.hashers.PBKDF2PasswordHasher')
```

Comment:
Maybe add a comment that Zulip was originally on PBKDF2 so we need it for compatibility, but we're now defaulting to Argon2?